test/cli-validator/tests/expectedOutput.js

```diff
@@ -263,4 +263,44 @@ describe('test expected output - OpenAPI 3', function() {
       'Operations must have a non-empty `operationId`.'
     );
   });
+
+  it('should produce a warning for properties with the same name but different type', async function() {
+    // set a variable to store text intercepted from stdout
+    const capturedText = [];
+
+    // this variable intercepts incoming text and pushes it into capturedText
+    const unhookIntercept = intercept(function(txt) {
+      capturedText.push(stripAnsiFrom(txt));
+      return '';
+    });
+
+    // set up mock user input
+    const program = {};
+    program.args = ['./test/cli-validator/mockFiles/duplicateProperties.json'];
+    program.default_mode = true;
+
+    const exitCode = await commandLineValidator(program);
+
+    // this stops the interception of output text
+    unhookIntercept();
+
+    expect(exitCode).toEqual(0);
+    expect(capturedText.length).toEqual(10);
+
+    expect(capturedText[1].trim()).toEqual('warnings');
+    expect(capturedText[2].trim()).toEqual(
+      'Message :   Properties person_prop and person_prop have the same name but different types'
+    );
+    expect(capturedText[3].trim()).toEqual(
+      'Path    :   paths./kid.post.requestBody.content.multipart/form-data.schema.properties.person_prop'
+    );
+    expect(capturedText[4].trim()).toEqual('Line    :   124');
+    expect(capturedText[6].trim()).toEqual(
+      'Message :   Properties name and name have the same name but different types'
+    );
+    expect(capturedText[7].trim()).toEqual(
+      'Path    :   components.schemas.kid.properties.name'
+    );
+    expect(capturedText[8].trim()).toEqual('Line    :   179');
+  });
```

Comment:
Since your feature is a validation, it should not be tested here. It should have its own unit test. See the other plugin tests for examples